react/features/toolbox/components/web/Toolbox.js

```diff
@@ -940,6 +947,15 @@ class Toolbox extends Component<Props, State> {
         return !this.props._isProfileDisabled && this._shouldShowButton('profile');
     }
 
+    /**
+     * Returns true if the camera toggle button is visible and false otherwise.
+     *
+     * @returns {boolean}
+     */
+    _isToggleCameraVisible() {
```

Comment:
We could put this check inside the button, setting the `visible` prop. As we do on mobile for all buttons.